Failed $make build

```
$ make setup
$ make build       
go build
main.go:11:2: cannot find package "golang.org/x/oauth2" in any of:
    /usr/local/Cellar/go/1.5.1/libexec/src/golang.org/x/oauth2 (from $GOROOT)
    /Users/t-kajinuma/src/golang.org/x/oauth2 (from $GOPATH)
make: *** [build] Error 1
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,7 +2,6 @@ language: go
 
 before_install:
   - make setup
-  - go get golang.org/x/oauth2
 
 script:
  - make travis

--- a/Makefile
+++ b/Makefile
@@ -2,6 +2,7 @@ setup:
 	go get -u github.com/google/go-github/github
 	go get -u github.com/jessevdk/go-flags
 	go get -u github.com/joho/godotenv
+	go get -u golang.org/x/oauth2
 
 build:
 	go build

